polldaddy-shortcode.php

```diff
@@ -178,6 +178,10 @@ function polldaddy_shortcode( $atts ) {
 			}
 		} elseif ( intval( $poll ) > 0 ) { //poll embed
 
+			if ( cs_is_amp_page() ) {
+				return sprintf( '<amp-iframe src="https://poll.fm/%d/embed" frameborder="0" height="400" layout="fixed-height" width="auto" sandbox="allow-scripts allow-same-origin" style="height: 400px; --loader-delay-offset:406ms !important;" i-amphtml-layout="fixed-height"></amp-iframe>', $poll );
```

Comment:
I feel like we had this conversation about surveys, but any way we can have the amp-iframe autoresize? It gets cut off:

<img width="830" alt="Screen Shot 2020-09-14 at 9 35 45 AM" src="https://user-images.githubusercontent.com/789137/93106623-c2e6df80-f66d-11ea-834b-579b6d0db5dc.png">

Reply:
Looks like there is support for resizing... involves changes to the `amp-iframe` and the content it is loading.
https://amp.dev/documentation/examples/components/amp-iframe/

I'll give this a try and see how it works.

Reply:
@roundhill thanks for asking about this again. I was able to add support to the poll iframe.

Might be able to do the same with our surveys. Just want to get you to test it on polls to make sure I'm on the right track 😄